cmd/ltctl/main.go

```diff
@@ -68,6 +68,18 @@ func RunStopCmdF(cmd *cobra.Command, args []string) error {
 	return t.StopCoordinator()
 }
 
+func RunSSHListCmdF(cmd *cobra.Command, args []string) error {
+	t := terraform.New(nil)
+	names, err := t.AppAndAgentNames()
```

Comment:
I think it may be better if we do the slice creation in this function itself. And not add one-off business logic methods to the package. What do you think of exposing the `terraformOutput` struct and the `getOutput` method ? It seems like something consumers of the package can benefit from, if they want to know the details of the deployment.

In future, if we can to have heuristics of recommending machine configs, exposing the getOutput method will come in handy.

Reply:
Sounds right, should we submit a separate PR for that change?

Reply:
We can do it in this PR itself. No need to unnecessarily add cruft to the package.